documentation.html

```diff
@@ -1114,11 +1120,554 @@ <h4 class="component-heading">Lists</h4>
+            class="code-example"
+            sandbox="allow-scripts allow-same-origin"
+            loading="lazy"
+          >
+          </iframe>
+        </section>
+        <!--  Slider Section End   -->
+
+        <!--  Forms Section Start   -->
+        <section class="component-section">
+          <h4 class="component-heading">Forms</h4>
+          <p class="component-sub-heading">
+            Forms are used to take the information from the user.
+          </p>
+          <p class="component-sub-heading-bold">Example of Forms</p>
+          <div
+            class="component-example flex-row flex-row-center-horizontal-vertical"
+          >
+            <form class="authentication-form">
+              <h2
+                class="font-large weight-semi-bold authentication-form-heading"
+              >
+                <span class="logo-text font-large">FiT</span>BiT
+              </h2>
+              <p
+                class="font-medium-large weight-semi-bold authentication-form-sub-heading"
+              >
+                LOGIN INTO YOUR ACCOUNT
+              </p>
+              <div class="form-input-box">
+                <input
+                  type="email"
+                  placeholder="Email Address"
+                  class="font-medium"
+                />
+                <i class="fa fa-envelope-o" aria-hidden="true"></i>
+              </div>
+              <div class="form-input-box">
+                <input
+                  type="password"
+                  placeholder="Password"
+                  class="font-medium"
+                />
+                <i class="fa fa-eye" aria-hidden="true"></i>
+              </div>
+              <div class="checkbox-container">
+                <span>
+                  <input type="checkbox" value="Remember Box" />
+                  <label for="Remember Box" class="font-medium"
+                    >Remember Me</label
+                  >
+                </span>
+                <a
+                  class="forgot-password-btn font-medium"
+                  href="forgotpassword.html"
+                >
+                  Forgot Password
+                </a>
+              </div>
+              <button class="button button-primary form-cta-button">
+                Login
+              </button>
+              <a class="button button-outline-primary form-cta-button" href="#">
+                SignUp With Us
```

Comment:
Just a suggestion to be written as Sign Up instead.

Reply:
Done